tor.nimble

```diff
@@ -1,10 +1,10 @@
 # Package
 
-version       = "0.1.0"
-author        = "Federico Ceratto"
+version       = "1.1.0"
+author        = "Federico Ceratto and KittyTechno"
 description   = "Tor helper library"
 license       = "MPL-2.0"
 
 # Dependencies
 
-requires "nim >= 0.20.0"
+requires "nim >= 1.2.6"
```

Comment:
Why this version bump? Let's use 1.0.0 unless there are real compatibility issues

Reply:
I put the version bump, because thats the current version of nim-lang I was developing in.

Reply:
Thats the current stable version of nim-lang, and is the current version I'm using.